src/main/java/pages/RegistrationPage.java

```diff
@@ -35,17 +38,37 @@ public void validRegistration (String firstName, String lastName, String email,
         clickButton(continue_btn);
     }
 
-    public void userLogout() throws InterruptedException {
+    public void openMyAccountMenu(){
+        clickButton(myAccount_btn);
+    }
+
+    public void clickOnLogout(){
         clickButton(logout_Btn);
     }
 
-    public void InvalidRegistration (String firstName, String lastName, String telephone,
-                                   String password) throws InterruptedException {
+    public void InvalidRegistrationWithFN_LN (String firstName, String lastName) throws InterruptedException {
```

Comment:
you can make one method with deferent tests, now you are repeating the same code over and over
just write the method one time and change the parameter in the test.
make in the same class annotation @test 3 or 4 times and with each one change the parameter of the test data as you want